### PR TITLE
style(docs): change the example mask and overlay z-index

### PR DESCRIPTION
**Pull request by btea**
Please make sure these boxes are checked before submitting your PR, thank you!

- [ ] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [ ] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

image
image

### CODE REVIEW

File: docs/.vitepress/vitepress/styles/css-vars.scss [hunk 1]
```diff
@@ -55,7 +55,7 @@
   --content-min-width: 16rem;
   --content-max-width: 48rem;
 
-  --nav-z-index: 10;
+  --nav-z-index: 10000;
```
Comment (ryuhangyeong) [inline docs/.vitepress/vitepress/styles/css-vars.scss:58]: If the sidebar is open, an exception might occur!

image

Comment (btea) [reply docs/.vitepress/vitepress/styles/css-vars.scss:58]: Thanks for your careful. There is also a problem when opening the dialog.

Comment (ryuhangyeong) [reply docs/.vitepress/vitepress/styles/css-vars.scss:58]: Thinking about it, I thought that changing the z-index of .el-table-v2__overlay in the element-plus documentation might have less side effects. What do you think about it?

Changing the z-index of a component will affect the user.

image

Comment (btea) [reply docs/.vitepress/vitepress/styles/css-vars.scss:58]: You are right, totally agree.

Comment (btea) [reply docs/.vitepress/vitepress/styles/css-vars.scss]: I have updated it, please review it again, thank you

**Review by ryuhangyeong — APPROVED** [review]

**Comment by ryuhangyeong** [comment]
Thank you for your effort!

**Pull request merged**